### ISSUE TITLE
[P2][Sourcing][Accessibility]`fdp-table` Missing tooltip for row selection

### ISSUE THREAD

**Issue by CSENoni** [SAP/fundamental-ngx#10055]
#### Is this a bug, enhancement, or feature request?

Bug

#### Briefly describe your proposal.

- From https://ui5.sap.com//#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.Selection, for both single and multiple selection, when we hover on first selection box, the cursor will be in pointer mode and then switch to regular mode with tooltip. In current platform table, the behavior is different on platform table and with no tooltip.

https://github.com/SAP/fundamental-ngx/assets/22842279/96b73b0e-515b-4ade-ac68-b3f4da943e97

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

latest or previous

#### If this is a bug, please provide steps for reproducing it.

Using https://ui5.sap.com//#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.Selection for reference.

**Comment by CSENoni** [comment]
@fkolar 

2. Maybe the the word "missing" I use here is a little bit aggressive. I should have typed "do not have" instead. I apologize if it caused any confusing. You once mentioned about this in https://github.com/SAP/fundamental-ngx/issues/9562:

> selection only happens on the level of 1st column (we should have behavior for the selection like UI5 so you can click on any column to select a row -Row , RowSelector , Row only) - but this would be different issue

3. https://fundamental-ngx.netlify.app/#/platform/table/row-selection for Single Row Selection.

I will change the issue type to enhancement.

**Comment by fkolar** [comment]
I can replicate now with the row-selection example. The white border is not definitely not correct. Unfortunately, none of the UI5 demos works today, not even specs is accessible. 
Can you ping me offline  - I am not sure who is CSENoni

**Comment by CSENoni** [comment]
Yes @fkolar, I will separate the second and fourth case into separate issue. Thanks!

**Comment by fkolar** [comment]
So this issue about a first  - missing tooltip for row selection

**Comment by CSENoni** [comment]
Tooltip is showing now for single and multiple selection in `0.43.8`. Thanks!
